suite/synctools/convert_unit_tests.py

```diff
@@ -38,7 +41,7 @@ def main():
             out_file = os.path.join(args.output, os.path.basename(test_file) + ".cs")
             print("  - Writting file: {}".format(out_file))
             with open(out_file, "w") as f:
-                f.write("# CS_ARCH_ARM64, 0, None\n")
+                f.write("# {}, {}, {}\n".format(args.arch, args.mode, args.opt))
```

Comment:
AArch64 is simple because it has only 1 mode, but other architectures may have more than that. for example ARM may has THUMB, or ARM mode, combining with V8 mode etc. you can try this on MC/ARM/, and see how it goes?

Reply:
I see what you are saying, this script will need to get much more complex to handle all the arch/modes